src/translations/main_en.ts

```diff
@@ -497,6 +497,10 @@ p, li { white-space: pre-wrap; }
 &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Thank you for your continuous support!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
+        <source>You can also remove folder with keystroke on Delete or Backspace key</source>
```

Comment:
```suggestion
        <source>You can also remove folders with Delete or Backspace</source>
```
I'm not a native speaker, but I think this sounds a bit more natural. What do you think?

Reply:
Yes, it seems more natural to me too! I'll adapt the French version to be more similar

Reply:
Perfect, thank you again! ❤️
Looks all good from my side.